modules/database/src/adapters/sequelize-adapter/index.ts

```diff
@@ -255,15 +256,24 @@ export class SequelizeAdapter extends DatabaseAdapter<SequelizeSchema> {
   }
 
   async ensureConnected() {
-    return this.sequelize
-      .authenticate()
-      .then(() => {
+    let error;
+    for (let i = 0; i < 10; i++) {
+      i === 0
+        ? ConduitGrpcSdk.Logger.log('Attempting to connect to database...')
+        : ConduitGrpcSdk.Logger.log(`Connection failed. Retrying...`);
+      try {
+        await this.sequelize.authenticate();
         ConduitGrpcSdk.Logger.log('Sequelize connection established successfully');
         return;
-      })
-      .catch(err => {
-        ConduitGrpcSdk.Logger.error('Unable to connect to the database: ', err);
-        throw err;
-      });
+      } catch (err) {
+        error = err;
+        await sleep(2000);
```

Comment:
That's a huge timeout. In cases were a db engine container is first run slightly after bringing up `Database` we're probably going to end up needlessly delaying Conduit's initialization by 2-4 secs over the db engine barely not making it in time.